ledger-service/jwt/src/main/scala/com/digitalasset/jwt/JwtVerifier.scala

```diff
@@ -43,31 +43,74 @@ object JwtVerifier {
   }
 }
 
+final case class LeewayOptions(
+    leeway: Option[Long] = None,
+    expiresAt: Option[Long] = None,
+    issuedAt: Option[Long] = None,
+    notBefore: Option[Long] = None,
+)
+
+sealed trait Leeway {
+  def getVerifier(
+      algorithm: Algorithm,
+      mbLeewayOptions: Option[LeewayOptions] = None,
```

Comment:
there is one problem with Optional's in Hocon config - you do not have a way to disable leeway-options if by default they're set. 

I.e. when you have smth like this by default:
```
type = unsafe-jwt-hmac-256
leeway-options {
  leeway = 1
} 
```
you can override this configuration by another hocon file apart from setting other values. For this reason for some of this 
configurations I did introduce "optReaderEnabled/optWriterEnabled" adaptor. It adds `enabled` for optional configurations.
Then leeway options can be disabled.

Reply:
Thank you for already solving it.

Do you have a pointer to this problem?
I think that a comment should be added in the definition of optReader stating the problem.
I can add the comment so as not to open a new PR.

Reply:
I am not sure what we would do in this specific case there. As we might never set defaults to some concrete value - it could be fine to keep it as you implemented it. 

There is no description of this problem, it left in the original HOCON PR comments :)